lib/zendesk_apps_tools/command.rb

```diff
@@ -29,15 +29,37 @@ class Command < Thor
     subcommand 'bump', Bump
 
     desc 'new', 'Generate a new app'
+    method_option :'iframe-only', type: :boolean,
+                                  default: false,
+                                  desc: 'Create an iFrame Only app template',
+                                  aliases: '-i'
     def new
-      @author_name  = get_value_from_stdin("Enter this app author's name:\n", error_msg: 'Invalid name, try again:')
-      @author_email = get_value_from_stdin("Enter this app author's email:\n", valid_regex: /^.+@.+\..+$/, error_msg: 'Invalid email, try again:')
-      @author_url   = get_value_from_stdin("Enter this app author's url:\n", valid_regex: /^https?:\/\/.+$/, error_msg: 'Invalid url, try again:', allow_empty: true)
-      @app_name     = get_value_from_stdin("Enter a name for this new app:\n", error_msg: 'Invalid app name, try again:')
+      enter = ->(variable) { "Enter this app author's #{variable}:\n" }
+      invalid = ->(variable) { "Invalid #{variable}, try again:" }
+      @author_name  = get_value_from_stdin(enter.call('name'),
+                                           error_msg: invalid.call('name'))
+      @author_email = get_value_from_stdin(enter.call('email'),
+                                           valid_regex: /^.+@.+\..+$/,
+                                           error_msg: invalid.call('email'))
+      @author_url   = get_value_from_stdin(enter.call('url'),
+                                           valid_regex: %r{^https?://.+$},
+                                           error_msg: invalid.call('url'),
+                                           allow_empty: true)
+      @app_name     = get_value_from_stdin("Enter a name for this new app:\n",
+                                           error_msg: invalid.call('app name'))
+
+      @iframe_location = if options[:'iframe-only']
+                           get_value_from_stdin("Enter your iFrame URI:\n", allow_empty: true) || 'assets/iframe.html'
+                         else
+                           '_legacy'
+                         end
 
       prompt_new_app_dir
 
-      directory('app_template', @app_dir)
+      skeleton = options[:'iframe-only'] ? 'app_template_iframe' : 'app_template'
+      is_default_iframe = options[:'iframe-only'] && @iframe_location != 'assets/iframe.html'
+      directory_options = is_default_iframe ? {} : { exclude_pattern: /iframe.html/ }
```

Comment:
I think this would make more sense like this:
```
is_custom_iframe = options[:'iframe-only'] && @iframe_location != 'assets/iframe.html'
directory_options = is_custom_iframe ? { exclude_pattern: /iframe.html/ } : {}
```